niCHARTPipelines/Structural.py

```diff
@@ -3,24 +3,59 @@
 import os,  shutil
 
 # from . import DeepMRSegInterface
-import nnUNetInterface
-import MaskImageInterface
-import ROIRelabelInterface
-import CalculateROIVolumeInterface
+from niCHARTPipelines import nnUNetInterface
+from niCHARTPipelines import MaskImageInterface
+from niCHARTPipelines import ROIRelabelInterface
+from niCHARTPipelines import CalculateROIVolumeInterface
 
-def run_structural_pipeline(inDir,DLICVmdl,DLMUSEmdl,outDir, MuseMappingFile,scanID,roiMappingsFile):
+def run_structural_pipeline(inDir,
+                            DLICVmdl_path,
+                            DLMUSEmdl_path,
```

Comment:
Shouldn't these be removed since both models are now passed via another param?

Reply:
My thinking here was to be futureproof, let's say if we train a nnUNetV2 model for one but not the other steps.